src/psbt.test.js

```diff
@@ -1,13 +1,22 @@
 import {
+  addSignaturesToPSBT, autoLoadPSBT,
```

Comment:
can these be split into two lines just to keep consistent? (grrr, prettier would've done this for you I'm pretty sure)